pkg/appgw/appgw_test.go

```diff
@@ -237,7 +237,7 @@ var _ = Describe("Tests `appgw.ConfigBuilder`", func() {
 	defaultBackendHTTPSettingsChecker := func(appGW *n.ApplicationGatewayPropertiesFormat) {
 		expectedBackend := &ingress.Spec.Rules[0].IngressRuleValue.HTTP.Paths[0].Backend
 		probeID := appGwIdentifier.probeID(generateProbeName(expectedBackend.Service.Name, serviceBackendPortToStr(expectedBackend.Service.Port), ingress))
-		httpSettingsName := generateHTTPSettingsName(generateBackendID(ingress, nil, nil, expectedBackend).serviceFullName(), fmt.Sprintf("%d", servicePort), Port(backendPort), ingress.Name)
+		httpSettingsName := generateHTTPSettingsName(generateBackendID(ingress, nil, nil, expectedBackend, expectedBackend.Service.Name).serviceFullName(), fmt.Sprintf("%d", servicePort), Port(backendPort), ingress.Name)
```

Comment:
`serviceFullName` returns `fmt.Sprintf("%v-%v", s.Namespace, s.Name)`

Reply:
can you append the namespace as well ?

Reply:
i believe this is still correct as we have only added an extra parameter to generatedBackendID. `serviceFullName` is still called but generateBackendID only uses the serviceName, as we have provided here.